### ISSUE TITLE
Color picker

### ISSUE THREAD

**Issue by timsueberkrueb** [timsueberkrueb/webber#8]

**Comment by gbdomubpkm** [comment]
If you are talking about a color selector for the home screen (is it that ?)  instead of manually entering a hex code, that would indeed be more convenient.

**Comment by timsueberkrueb** [comment]
Yeah, though this has very low priority. Contributions highly welcome, mentoring available!

**Comment by gbdomubpkm** [comment]
Yes i agree, it isn't a priority, it is indeed only for the confort of use. But maybe, at some point, @pavelprosto94 will be able to help or to give you advices as he used a similar mechanism i think in uHome:
![screenshot20210828_164558915](https://user-images.githubusercontent.com/50362604/131222361-2bfbd1c2-c258-49e0-923e-672c220e6c3c.png)

**Comment by timsueberkrueb** [comment]
Uhh, thank you for the tip! This looks very nice. If their source code is GPL, it shouldn't be a problem to include it in Webber.
